google/ggrc-core
# [5OBJ] GGRC-8531 Clean HTML tags for 5OBJ and their Comments

# Dependencies

_**This PR should be merged into [GGRC-7200](https://github.com/google/ggrc-core/tree/GGRC-7200) branch which is the common one for all 5OBJ. PRs**_

This PR is `on hold` until the dependencies are merged:

- [x] #10615 (Move Contracts & Policies out into separate tables)
- [x] #10631 (Extend 5OBJ with Q-side fields)


# Issue description

For external models we need to clean attributes and comments from HTML tags.

# Steps to test the changes

1. Run the migration and make sure it pass successfully.
2. Make sure there are not any HTML tags left in 5OBJ _description_, _notes_, and _test_plan_ columns after the migration.

# Solution description

* Create `WithDefaultUpdateAttrMixin` mixin which defines default `update_attributes()` implementation for any `BaseConverter` subclass.

* Create converters for 5OBJ. Converters for Contracts and Policies sublcass `DirectivesConverter`, and converters for Objectives, Requirements, and Threats subclass `WithDefaultUpdateAttrMixin` and `BaseConverter`.

# Sanity checklist


# Migration checklist

# PR Review checklist
